cmd/internal/cli/apptainer.go

```diff
@@ -282,7 +288,7 @@ func setSylogMessageLevel() {
 	} else if silent {
 		level = -3
 	} else {
-		level = 1
+		level = sylog.Max(level, 1)
```

Comment:
Use `math.Max()` instead and delete the new function below.

Reply:
thanks! reflected in this commit
https://github.com/apptainer/apptainer/pull/1569/commits/1a2003474ceaa5144f602b234aa4bd811f3d336b